pom.xml

```diff
@@ -769,19 +769,13 @@
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>${maven-checkstyle-plugin.version}</version>
         <configuration>
-          <sourceDirectories>
-            <directory>${basedir}/src/main/java</directory>
-          </sourceDirectories>
-          <testSourceDirectories>
-            <directory>${basedir}/src/test/java</directory>
-          </testSourceDirectories>
           <configLocation>checkstyle.xml</configLocation>
+          <suppressionsLocation>checkstyle-suppressions.xml</suppressionsLocation>
           <encoding>UTF-8</encoding>
           <consoleOutput>true</consoleOutput>
-          <failsOnError>true</failsOnError>
+          <failsOnError>false</failsOnError>
```

Comment:
why do we need `false` option?

Reply:
This option does not matter, because there is no `<execution>` config in checkstyle.
If it has `<execution>` config, it will fail `mvn package` if checkstyle fails if set `true`.

Reply:
In origin configuration, we seems not to check test file import order. But if we have wrong import order in our main code, checkstyle will fail. In new configuration, we can check import order in test code, but checkstyle didn't fail. Is there anything wrong?

Reply:
> In origin configuration, we seems not to check test file import order. But if we have wrong import order in our main code, checkstyle will fail. In new configuration, we can check import order in test code, but checkstyle didn't fail. Is there anything wrong?

Can you give an example link to CI (before and after)?

Reply:
We can see your link
<img width="782" alt="企业微信截图_409b5f2d-3e53-490b-b59e-81a3fc3afdf6" src="https://user-images.githubusercontent.com/8159038/171576340-d2a2d5a8-4603-4409-986b-bef81281333d.png">
Although there some warning, but checkstyle didn't fail.

Reply:
This is because of checkstyle.xml

https://github.com/Tencent/Firestorm/blob/795f75a7030a4ac204aaef20f0f70caba1a39d0c/checkstyle.xml#L26

Reply:
We can turn it on while we are fixing checkstyle violations.

Reply:
How can I turn it on?

Reply:
> How can I turn it on?

Remove
https://github.com/Tencent/Firestorm/blob/795f75a7030a4ac204aaef20f0f70caba1a39d0c/checkstyle.xml#L26